### ISSUE TITLE
test-ballooning fails

### ISSUE THREAD

**Issue by dschwoerer** [boutproject/BOUT-dev#985]
Not sure why, error is:
Fail, maximum difference = 1.033736e+00

Branch: test-requirements

**Comment by dschwoerer** [comment]
Found the solution:
The "old method" is not supported any more.
Is this documented?
Should we check for something and throw an error?

```
[var0]  # Variable 0, using old method

scale = 1

xs_opt = 0  # constant

ys_opt = 1  # Gaussian
ys_s0 = 0.5  # Centered at middle of domain
ys_wd = 0.5  # Width

zs_opt = 2  # Sinusoidal
zs_mode = 2 # Mode number
zs_phase = 0
```

**Comment by ZedThree** [comment]
Fixed in #1005
